dp2/config.yml

```diff
@@ -1,18 +1,20 @@
-#WS CONFIGURATION
+# Pipeline's webservice host
 host: http://localhost
+# Pipeline's webservice port
 port: 8181
-ws_path: ws
-ws_timeup: 25
-#DP2 launch config 
-exec_line: ../../../../../../../daisy/pipeline-assembly/target/dev-launcher/bin/pipeline2
-
-local: true
-# ROBOT CONF
-client_key: clientid
-client_secret: supersecret
-#connection settings
+# Pipeline's webservice path, as in HOST:PORT/PATH (e.g. http://localhost:8181/ws)
+path: ws
+# Timeout for requests to the webservice in seconds
 timeout: 10
-#debug
-debug: false
-starting: true
+# Path of the DAISY Pipeline app or the pipeline launch script to use the webservice.
+# If empty or only containing 'DAISY Pipeline', the app is searched in the PATH
```

Comment:
Are paths resolved relative to this config file? If so, it's probably good to add a note here and in the help.

Reply:
Hi @bertfrees, I rechecked the code that resolves the path : if it is a relative path that is not registered in the PATH variable (that's the bit I added in the path retrieval), it is evaluated as relative to the "executable folder" in the config.go functions that resolve it.

From the doc of the function, this is the folder that contains the running executable (but i'm not sure if this resolve symlinked executables)

https://github.com/NPavie/pipeline-cli-go/blob/88ce4298e1afb356ec61640ba97b013f2cf703ab/cli/config.go#L219-L242

Reply:
Thinking about it, it might be a good idea to change this logic and indeed use a "config relative" evaluation of the path if the path is not absolute or if it is not an executable exposed in the PATH environment variable

Reply:
Yes, I too think that makes sense.

Reply:
@NPavie Will you make that change?

Reply:
i'm on it

Reply:
thanks!